cypress/integration/splitButton.feature

```diff
@@ -0,0 +1,68 @@
+Feature: Split Button component
+  I want to change Split Button component properties
+
+  Background: Open Split Button component page classic
+    Given I open "Split Button" component page classic
+
+  @positive
+  Scenario Outline: I select as to <as>
+    When I select as to "<as>"
+    Then Button background color is "<color>"
```

Comment:
if the statement will start from button word, it should start from small letter, but if U want to start statement from Split Button -it should start with Capital character

Reply:
it is capial letter because step is reused